apps/linows/src/css/components/superactions.css

```
@@ -12,23 +12,47 @@ html.controls-open #hint-bar {
 }
 
 .control-strip {
+    /* A beat above the bento, on top of the bar's own gap (macOS
+     * Launchpad.outerTopPadding). The strip is a different kind of thing from
+     * the bar, so the seam wants more air than the tile-to-tile gaps. */
+    --ctl-outer-top: 8px;
     flex: 1;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)